modules/admin-ui/package-lock.json

```diff
@@ -2339,8 +2339,7 @@
         "ansi-regex": {
```

Comment:
You probably did not want to submit this

Reply:
Concur with removing this.